ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -32,12 +32,20 @@ class TabBarViewDemoController: DemoController {
         return createButton(title: "-", action: #selector(decrementBadgeNumbers))
     }()
 
+    private lazy var homeItem: TabBarItem = {
```

Comment:
What's the rationale behind this being a lazy var in the TabBarViewDemoController while the SideTabBarDemoController is just a constant (let)?

Reply:
Ohh!! My bad!! Now, I made lazy var at both places because there is no need to make them a constant.